lib/claim_token/version.rb

```diff
@@ -1,3 +1,3 @@
 module ClaimToken
-  VERSION = "0.1.0"
+  VERSION = "0.1.1".freeze
```

Comment:
Do we actually need to specify freeze now ? I though this was being automatically done by ruby ?

Reply:
According to this stack overflow discussion you should do this or add the frozen string literal magic comment until Ruby 3, in which it'd do this by default https://stackoverflow.com/a/37799399